components/service/pocket/src/main/java/mozilla/components/service/pocket/stories/PocketRecommendationsRepository.kt

```diff
@@ -17,21 +18,21 @@ import org.json.JSONObject
 import java.io.File
 
 @VisibleForTesting internal const val STORIES_FILE_NAME = "pocket_recommended.stories"
-@VisibleForTesting internal const val JSON_KEY_ARRAY_STORIES = "list"
+@VisibleForTesting internal const val JSON_KEY_ARRAY_STORIES = "recommendations"
 
 /**
  * Wrapper over our local database.
  * Allows for easy CRUD operations.
  */
-internal class PocketRecommendationsRepository(private val context: Context) {
+internal object PocketRecommendationsRepository {
```

Comment:
(Same questions as for the use cases)

Reply:
Reverted to use the same previous implementation.